Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/services/QuestionService.java

```diff
@@ -1,10 +1,16 @@
 package services;
 
-import entity.Question;
+import hibernate.entity.Question;
+import hibernate.entity.Quiz;
+import org.springframework.stereotype.Service;
 
 import java.util.Set;
 
+@Service
 public class QuestionService {
+
+    public static Quiz currentQuiz;
```

Comment:
Static quiz? What for?

Reply:
This is the current questionnaire used. It is initialized after selecting from the list of questionnaires. After that, the current questionnaire is used when adding questions.
What's wrong with static, I do not understand